cluster/src/main/java/org/apache/iotdb/cluster/client/sync/SyncClientAdaptor.java

```diff
@@ -165,14 +165,14 @@ public static Long querySingleSeries(
   }
 
   public static List<String> getNodeList(
```

Comment:
Because these two methods use `SingleSeriesQueryRequest`, I have added `raftId` into this request. So it's unnecessary to add another parameter in methods.